wordcount.c

```diff
@@ -0,0 +1,95 @@
+/* Jared Wasserman -- wordcount.c */
```

Comment:
nice job! the only thing is that you forgot to take tabs into consideration, so it counted fewer words than it should have!